src/main/groovy/life/qbic/datamodel/datasets/datastructure/folders/nanopore/Fast5FailFolder.groovy

```diff
@@ -0,0 +1,34 @@
+package life.qbic.datamodel.datasets.datastructure.folders.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+
+/**
+ * A special case of a DataFolder, its name is always fast5_fail.
+ *
+ * Its children field contains either a list of type List<Fast5Files> or List<Fast5Folder>
+ *
+ * @author: Sven Fillinger
+ */
+class Fast5FailFolder extends DataFolder {
+
+    final private static String NAME = "fast5_fail"
+
+    final private static String NAME_SCHEMA = /fast5_fail/
+
+    protected Fast5FailFolder() {}
+
+    protected Fast5FailFolder(String relativePath, List children) {
+        super(NAME, relativePath, children)
+        validateName()
```

Comment:
```suggestion
        if (!isValidateName(name)) {
            throw new IllegalArgumentException("Name must match the QBiC barcode schema!")
        }
```

Reply:
Ah, negated boolean are always easy to misread. I try to avoid it. Again here,, you violate the "tell, don't ask" principle. Just call `validate()` and if it is valid, the object gets instantiated. Otherwise you end up in nested if/else conditions, especially in the constructor this something we should try to avoid if possible.

Reply:
I personally find it harder to read since you have to look up what actually happens if the name is invalid. Maybe rename the method to `failForInvalidName()` or something that indicates: "This method might blow up"